packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -83,11 +83,21 @@ export const getOnboardingStatus = {
   }
 }
 
-export const buildContactsQueryByIds = (ids = []) => ({
+export const buildContactsQueryByIds = (ids = [], enabled = true) => ({
   definition: () => Q(CONTACTS_DOCTYPE).getByIds(ids),
   options: {
     as: `${CONTACTS_DOCTYPE}/${ids.join('')}`,
-    fetchPolicy: defaultFetchPolicy
+    fetchPolicy: defaultFetchPolicy,
+    enabled
+  }
+})
+
+export const buildContactsQuery = (enabled = true) => ({
+  definition: () => Q(CONTACTS_DOCTYPE),
```

Comment:
The default 100 pagination might considerably slow down the query if there are a lot of contacts: for 5K contacts, this will fire 50 queries, vs 5 by adding a `.limitBy(1000)`.

Reply:
exact, je corrige